check-status.sh

```
@@ -0,0 +1,58 @@
+    data = json.load(f)
+    metrics = data['metrics']['_totals']
+    print(f'     HIGH: {metrics[\"SEVERITY.HIGH\"]} MEDIUM: {metrics[\"SEVERITY.MEDIUM\"]} LOW: {metrics[\"SEVERITY.LOW\"]}')
+"
+echo "   Frontend Audit:"
+cd frontend
+yarn audit --level moderate 2>&1 | grep "vulnerabilities found" | sed 's/^/     /'
+cd ..
+
+echo ""
+echo "✅ 4. BUILDS"
+echo "   Frontend build artifacts:" $(ls frontend/dist/ 2>/dev/null | wc -l) "files"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)